tests/01_microtest.R

```diff
@@ -1,18 +1,29 @@
 # detach(package:MoBiToolboxForR)
+# install.packages("C:/workspace/sw/MoBiToolBoxForR_7.1.0.zip", repos = NULL, type = "win.binary", lib = "C:/workspace/sw/TBforRVERSIONS/7.1.0/")
```

Comment:
Sure. This was an example. There are still many absolute paths in this file which would make reuse of those tests on another dev computer very time consuming. If absolute path cannot be avoided, suggestion is to create a variable at top of file with clear indications that this value have to be changed on a 'per user' basis.